python-bindings/overview_article/tasks.py

```diff
@@ -1,48 +1,95 @@
 """ Task definitions for invoke command line utility for python bindings
-    overview article. """
+    overview article.
+
+    If anything is causing trouble get in touch at
+    Znunu @ github or VimVim @ fcZBB2v @ discord"""
 import cffi
 import invoke
 import pathlib
+import sys
+import os
+import re
+
+on_win = sys.platform.startswith("win")
 
 
 @invoke.task
 def clean(c):
     """ Remove any built objects """
-    for pattern in ["*.o", "*.so", "cffi_example* cython_wrapper.cpp"]:
-        c.run("rm -rf {}".format(pattern))
+    for pattern in (
+        "*.o",
+        "*.so",
+        "*.obj",
+        "*.dll",
+        "*.exp",
+        "*.lib",
+        "*.pyx",
+        "*.pyd",
+        "cffi_example*",
+        "cython_wrapper.cpp",
+    ):
+        if on_win:
+            c.run("del {} >nul 2>&1".format(pattern))
+        else:
+            c.run("rm -rf {}".format(pattern))
+    if on_win:
+        c.run("rmdir /s /q Release >nul 2>&1")
```

Comment:
That's nice.

Reply:
Maybe I'm missing something but could we replace those delete/rm commands with something Python-native that works across platforms? 

E.g. https://stackoverflow.com/questions/6996603/how-to-delete-a-file-or-folder

Reply:
Doh!  Can't believe I missed this.   Thanks, @dbader